python_modules/dagster/dagster_tests/asset_defs_tests/test_assets_job.py

```diff
@@ -2352,7 +2352,7 @@ def test_build_subset_job_errors(job_selection, use_multi, expected_error):
     if expected_error:
         expected_class, expected_message = expected_error
         with pytest.raises(expected_class, match=expected_message):
-            Definitions(assets=assets, jobs=[asset_job])
+            Definitions(assets=assets, jobs=[asset_job]).get_all_job_defs()
```

Comment:
note that I had to add this for certain types of validation errors to still get bubbled up - they will still be surfaced in 'dagster dev' though (the gRPC server always loads all definitions, even if they are lazy)

Reply:
maybe leave a comment